hw2/02-search.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>02 - Search</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
+    <script defer src="02-data.js"></script>
+    <script defer src="02-search.js"></script>
+  </head>
+  <body class="bg-dark">
+    <main class="container mt-5">
+      <section class="bg-light border rounded p-4">
+        <h1 class="text-center card-title mb-4">Exercise 02 - Search</h1>
+        <div class="mb-3 d-flex justify-content-center">
+          <label for="userInput"></label>
+          <input type="text" id="userInput" />
+          <input
+            class="btn-primary text-light border rounded"
+            type="button"
```

Comment:
Wave extension flagged this for contrast error. This can be fixed by using class="btn btn-primary"

Reply:
Fixed!